ext/nnstreamer/tensor_filter/tensor_filter_tvm.cc

```diff
@@ -320,10 +320,13 @@ tvm_subplugin::invoke (const GstTensorMemory *input, GstTensorMemory *output)
   tvm::runtime::NDArray tensor;
 
   /* read functions */
-  tvm::runtime::PackedFunc set_input = gmod.GetFunction ("set_input");
+  tvm::runtime::PackedFunc set_input = gmod.GetFunction ("set_input_zero_copy");
   if (set_input == nullptr) {
-    cleanup ();
-    throw std::runtime_error ("Packed function `set_input` not defined in model");
+    set_input = gmod.GetFunction ("set_input");
```

Comment:
Is it okay to _implicitly_ alternate `set_input_zero_copy` to `set_input` in terms of memory management?

Reply:
Not sure.. I don't think of any other option other than this or throwing an error. But in normal case `set_input_zero_copy` is supported.

Reply:
IMO, it's better to throw here if we are not sure, or issue a warning :)

Reply:
I agree